modules/apps/digital-signature/digital-signature-test/src/testFunctional/macros/DigitalSignature.macro

```diff
@@ -340,12 +366,50 @@ definition {
 			locator1 = "DigitalSignatureEnvelopeSettings#ANY_DISABLED_BUTTON");
 	}
 
+	macro viewCreateDate {
+		WaitForElementPresent(locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_RECENT_CREATE_DATE");
+
+		AssertElementPresent(locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_RECENT_CREATE_DATE");
+	}
+
 	macro viewDigitalSignatureEnabled {
 		AssertElementPresent(
 			dsStatus = ${dsStatus},
 			locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_STATUS");
 	}
 
+	macro viewDigitalSignaturestatus {
+		AssertElementPresent(
+			dsStatus = ${viewStatus},
+			locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_STATUS");
+	}
+
+	macro viewEnabledDownload {
+		Click(
+			envelopeName = ${documentName},
+			locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_ELLIPSIS_BUTTON");
+
+		AssertElementPresent(locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_DOWNLOAD_BY_LIST");
+	}
+
+	macro viewEnvelopeDetails {
+		Click(
+			envelopeName = ${documentName},
+			locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_ENVELOPE_NAME");
+
+		AssertElementPresent(locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_ENVELOPE_ID");
+	}
+
+	macro viewMoreRecipient {
+		WaitForElementPresent(locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_RECIPIENTS_BADGE");
+
+		AssertElementPresent(locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_RECIPIENTS_BADGE");
+
+		DigitalSignature.deleteEnvelope(envelopeName = ${documentName});
+
+		AssertElementNotPresent(locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_RECIPIENTS_BADGE");
```

Comment:
Could you remove these steps: `DigitalSignature.deleteEnvelope` and` AssertElementNotPresent` due to  these stepes don't belong to the view recipient process? You don't need create another macro for these ones